raglweb/__init__.py

```diff
@@ -284,11 +285,39 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
+    """Convert the human readable division title into a prefix for use in the filename."""
+    words = division_title.upper().split()
+    division_prefix = words[0]
+    if division_prefix.endswith('S'):
+        division_prefix = division_prefix[:-1]
+    division_prefix += ''.join([word[0] for word in words[1:]])
+    season = _cfg['season']
+    prefix = f'RAGL-S{season:02d}-{division_prefix}-'
+    return prefix
 
 @app.route('/replay/<replay_hash>')
 def replay(replay_hash):
     db = _db_get()
-    cur = db.execute('SELECT filename FROM outcomes WHERE hash=:hash', dict(hash=replay_hash))
-    fullpath = cur.fetchone()['filename']
+    cur = db.execute('''
+        SELECT
+            filename,
+            p0.division as p0_division,
+            p1.division as p1_division,
```

Comment:
I just tested and you have a syntax error here: you need to drop the trailing comma.

Reply:
Thanks - fixed.